### PR TITLE
fix resizing bug after pasting object

### LINKED ISSUE #2281: Resizing of 'the selection just after pasted' behaves oddly
This is a regression with the commit 6e5469874290871c501f14dbbca36099e96231d8 .

**Affects versions :**
 - OS: Arch Linux
 - (Linux only) Desktop environment: X11
 - Which version of libgtk do you use: 3.24.23
 - Version of Xournal++: master (after 6e5469874290871c501f14dbbca36099e96231d8 )
 - Installation method: built from source

**Describe the bug**
When a copied object is pasted, it appears as a selected object.
But if you click the corner or the edge to resize it, the object
will jump to another place or disappear.

**To Reproduce**
1. Create a stroke. Call the object AAA.
2. Select AAA and copy it to the clipboard.
3. Paste the object in the clipboard. Call the appeared object BBB.
4. BBB appears as a selected object. Click one of the corner of it with a pen.

**Additional context**
1. If you click the corner with a mouse, this bug does't appears.

2. If you use LaTeX tool, the object generated by LaTeX  also
appears as a selected object. This object also gets this bug.

3. If you apply the commit 6f29f1cc85a576461d927f1ce347da4293179d36 ,
this bug seems to be avoided if the paste is the first time.
But if you try another paste succesively, it gets this bug again.

### PR DESCRIPTION
**Pull request by rolandlo**
Fixes #2281

### CODE REVIEW

File: src/control/tools/EditSelectionContents.cpp [hunk 1]
```diff
@@ -523,13 +522,7 @@ void EditSelectionContents::serialize(ObjectOutputStream& out) {
...
     in.readObject("EditSelectionContents");
 
-    double originalWidth = in.readDouble();
-    double originalHeight = in.readDouble();
-
-    double originalX = in.readDouble();
-    double originalY = in.readDouble();
-
-    this->originalBounds = Rectangle<double>{originalX, originalY, originalWidth, originalHeight};
+    this->originalBounds = Rectangle<double>{in.readDouble(), in.readDouble(), in.readDouble(), in.readDouble()};
```
Comment (Technius) [inline src/control/tools/EditSelectionContents.cpp:525]: This is less readible; side effect order should be explicit.

**Review by Technius — APPROVED** [review]

File: src/control/tools/EditSelectionContents.cpp (hunk 1 above)
Comment (rolandlo) [reply src/control/tools/EditSelectionContents.cpp:525]: Ok, I will change it back then.

**Comment by rolandlo** [comment]
Thanks for looking at it @Technius. I have made the effect order explicit again. Shall I squash and merge?

**Comment by Technius** [comment]
I'll did it for you to save time ;)

**Pull request merged**